### PR TITLE
cleaned up filesystem

### PR DESCRIPTION
**Pull request by chriseld**
put css and js files into their own subfolders, removed sass import (for now at least)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,13 +4,13 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/bulma@0.8.0/css/bulma.min.css">
-    <link rel="stylesheet" type="text/css" href="./assets/styles.css">
+    <link rel="stylesheet" type="text/css" href="./assets/css//styles.css">
```
Comment (jethan16) [inline index.html]: typo?

**Review by jethan16 — CHANGES_REQUESTED** [review]
typo

**Comment by chriseld** [comment]
fixed typo

**Review by jethan16 — APPROVED** [review]
looks good!

**Pull request merged**